README.md

```diff
@@ -251,6 +252,50 @@ snakemake \
 After successful execution of the workflow, results and logs will be found in
 the `results/` and `logs/` directories, respectively.
 
+### Expected output files
+
+Upon successful execution of _MIRFLOWZ_, the tool automatically removes all
+intermediate files generated during the process. The final output comprises:
+
+1. A SAM file containing alignments intersecting a pri-miR locus. These
+alignments intersect with extended start and/or end positions specified in the
+provided pri-miR annotations. Please note that they may not contribute to the
+final counting and will not appear in the final table.
+
+2. A SAM file containing alignments intersecting a miRNA locus. Similar to the
+previous file, these alignments intersect with extended start and/or end
+positions specified in the provided miRNA annotations. They may not contribute
+to the final counting and might be absent from the final table.
+
+3. A SAM file containing the uncollapsed set of alignments that contribute to
+the final counting.
+
+4. A BAM file containing the uncollapsed set of alignments contributing to the
+final counting and its corresponding index file (`bam..bai`).
+
+5. Table(s) containing the counting data from all libraries for (iso)miRs
+and/or pri-miRs. Each row corresponds to a miRNA species, and each column
+represents a sample library. Counting involves aggregating contributions from
+all alignments, calculated as the ratio of collapsed reads in th alignment to
+the number of hits (NH value).
+
+To retain all intermediate files, include --no-hooks in the workflow call.
+
+```bash
+snakemake \
+    --snakefile="path/to/Snakefile" \
+    --cores 4  \
+    --configfile="path/to/config.yaml" \
+    --use-conda \
+    --printshellcmds \
+    --rerun-incomplete \
+    --no-hooks \
+    --verbose
+```
+
+After successful execution of the workflow, the intermediate files will be
+found in the `results/inter_files` directory.
```

Comment:
I generally think it's good to name things explicitly. So perhaps rather name the directory `intermediate_files` or `intermediates`, rather than `inter_files`.